src/BoilerPlate/App.cpp

```diff
@@ -34,6 +36,33 @@ namespace Engine
 		CleanupSDL();
 	}
 
+	void App::CreateAsteroid(int amount)
+	{
+		srand(time(NULL));
+
+		int currentSize;
+		float xCoordinate;
+		float yCoordinate;
+		float orientation;
+		int signChanger = 1; //Used to ensure asteroids will have their initial position in both positive and negative sides of the screen
+		
+		for (int i = 0; i < amount; i++)
+		{
+			currentSize = rand() % 3;
```

Comment:
If you're using an enumeration for the `Asteroid` sizes you would just need to reference the one with the biggest value, assuming your enumeration values are sequential.